torch/testing/_internal/common_utils.py

```diff
@@ -2011,10 +2017,11 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
             result = torch.rand(size, device=device, dtype=dtype) * span + low
     else:
         assert dtype in complex_types()
```

Comment:
Couldn't we have complex dtypes as an extra `elif` clause? The author of this line clearly wanted to future proof `make_tensor` in case there is a new dtype that is not covered by the current branches. In that case the raised `AssertionError` is not really helpful. If we put complex in a separate branch, we could raise an expressive error in the `else` branch.

Reply:
Agreed, thanks for pointing this out. Done: https://github.com/pytorch/pytorch/pull/61108/commits/bafe03bdf63b33abd7565e9560bc4480fa8917de.